### ISSUE TITLE
error when opening "make chaff" dialog

### ISSUE THREAD

**Issue by az0** [bleachbit/bleachbit#606]
I cannot open the "make chaff" dialog from the application menu

```
Traceback (most recent call last):
  File "bleachbit\GUI.pyo", line 125, in cb_make_chaff
  File "bleachbit\GuiChaff.pyo", line 30, in <module>
  File "bleachbit\Chaff.pyo", line 22, in <module>
ImportError: No module named email.generator
```

system
```
BleachBit version 2.3
Git revision ccfb0cdc
GTK version 3.18.9
SQLite version 3.28.0
FSE = mbcs
locale.getdefaultlocale = ('es_ES', 'cp1252')
platform.platform = Windows-10-10.0.18362
sys.version = 2.7.16 (v2.7.16:413a49145e, Mar  4 2019, 01:30:55) [MSC v.1500 32 bit (Intel)]
win32com.shell.shell.IsUserAnAdmin() = False
```

This is the "log" branch in Git

Application is running in portable mode. 

Python is not installed.

**Comment by az0** [comment]
Seen _also_ on same Windows 10 system on rados-ms1 branch (Git revision 54351574)

**Comment by az0** [comment]
On fix branch with commit 77443af4, BleachBit GUI windows closes when chosing "make chaff"

**Comment by az0** [comment]
211e71ba80adeb07452f14ef28741a3bf5e2ffa2 fixed "No module named email.generator" and related errors. The next error about missing the markovify module
